js/scripts.js

```diff
@@ -4,5 +4,5 @@ var allNames = nameMale.concat(nameFemale);
 
 var newName = 'Marian';
 allNames.splice(0, 0, newName);
-    
```

Comment:
try to use method `indexOf` :) when element doesn't exist in array method indexOf will return you value `-1`